mongodb/intro/query-operators/objectid-intro.md

```diff
@@ -29,45 +28,6 @@ Here is an example of an `ObjectId` value:
 x = ObjectId("507f1f77bcf86cd799439011");
```

Comment:
if not too hard we can do something like this to show which bytes are which
```js
ObjectId("507f1f77bcf86cd799439011")
//              ^--^ ^--^ ^-----^
```

Reply:
I can't find the official information regarding which is which

Reply:
I tried to do it but I can't seem to find the information regarding which is which. I think we can leave it as is. If you have some official information I can use, send me I will make it

Reply:
after a bit of research I found this https://stackoverflow.com/a/58280221

I think it will be too hard to show what each character represents, so disregard this

Reply:
Yeah, sadly there is no simple way of doing this. Would be nice to add in the future tho